rpm/centos-7/docker-ce.spec

```diff
@@ -1,217 +0,0 @@
-Name: docker-ce
-Version: %{_version}
-Release: %{_release}%{?dist}
-Epoch: 3
-Summary: The open-source application container engine
-Group: Tools/Docker
-License: ASL 2.0
-Source0: engine.tgz
-Source1: cli.tgz
-URL: https://www.docker.com
-Vendor: Docker
-Packager: Docker <support@docker.com>
-
-# DWZ problem with multiple golang binary, see bug
-# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
-%global _dwz_low_mem_die_limit 0
```

Comment:
I was looking if we would still need lines like this in the new specs (as these were not in there).

but we don't need these files in the repository for that of course (we can compare with an older commit to check what's missing)